multiworld/envs/mujoco/cameras.py

```diff
@@ -18,7 +18,6 @@ def init(camera):
 
     return init
 
-
```

Comment:
Just a heads up: my pycharm is configured to leave two spaces between high-level definitions (like between functions and classes). I think it's quite nice. I don't feel strongly about it, but I'd rather not refactor everything so that there's only one line between each function

Reply:
oh no haha, i did that because it bothers me when there is more than one space between functions .-., ill fix that